docs/DeveloperGuide.md

```diff
@@ -550,234 +554,125 @@ Use case ends.
-    Use case ends.
+**MSS**
+*  Similar MSS to adding a company except user is now adding an internship.
 
 **Extensions**
 
- 1a. InternHunter detects an error in the input format. <br/>
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
-  Use case resumes from step 1.
+ 1a. Similar to extension 1a of adding a company.
 
  1b. InternHunter detects an invalid index. <br/>
   1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
   Use case resumes from step 1.
 
-#### Use case: UC08 - Add an application
-
-Precondition: User already has an existing list of internships in a company <br/>
-Guarantees: Adding of application is successful
-
-**MSS**
-
-1.  User requests to apply for an internship and provides the index and relevant details.
-2.  InternHunter adds the application to the list of applications and prompts the user to switch to the Applications
-tab to view the newly added application. <br/>
-    Use case ends.
-
-**Extensions**
-
- 1a. InternHunter detects an error in the input format. <br />
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br />
-  Use case resumes from step 1.
-
- 1b. InternHunter detects an invalid index. <br />
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br />
-  Use case resumes from step 1.
-
-#### Use case: UC09 - Delete an application
-
-Precondition: User already has an existing list of applications <br />
-Guarantees: Deletion of application is successful
-
-**MSS**
-
-1.  User requests to delete an application and provides the index.
-2.  InternHunter removes the application from the list of applications. <br />
-    Use case ends.
-
-**Extensions**
-
- 1a. InternHunter detects an error in the input format. <br />
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br />
-  Use case resumes from step 1.
-
- 1b. InternHunter detects an invalid index. <br />
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br />
-  Use case resumes from step 1.
-
-#### Use case: UC10 - Edit an application
+#### Use case: UC08 - Delete an internship
 
-Precondition: User already has an existing list of applications <br />
-Guarantees: Editing of application is successful
-
-**MSS**
-
-1.  User requests to edit the details of an application and inputs the index and details.
-2.  InternHunter edits the details of this application and the list is updated accordingly. <br />
-    Use case ends.
-
-**Extensions**
-
- 1a. InternHunter detects an error in the input format. <br />
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br />
-  Use case resumes from step 1.
+* Similar to deleting a company except user is deleting an internship
```

Comment:
I think this is a very neat way of reducing duplication in our use cases section! Whenever we reference other use cases like this, should we perhaps be a little more specific by providing the use case number and exact title? For example, writing `Similar to UC02 - Delete a company, except that the user is deleting an internship` instead? And perhaps we could even hyperlink the `UC02 - Delete a company` part?